lib/rubocop/cop/cop.rb

```diff
@@ -212,10 +217,16 @@ def reference_url
         (url.nil? || url.empty?) ? nil : url
       end
 
+      def details
+        details = cop_config && cop_config['Details']
```

Comment:
I have no idea - I just copied that from how the other similar methods were set up ...